src/test/java/com/ifactory/press/db/solr/spelling/suggest/MultiSuggesterTest.java

```diff
@@ -138,17 +158,18 @@ private void assertNoSuggestions() throws SolrServerException {
     }
 
     private void assertSuggestions() throws SolrServerException {
-        Suggestion suggestion = assertSuggestionCount("t", 8, "all");
+        Suggestion suggestion = assertSuggestionCount("t1", 5, "all");
         // TITLE occurs once in a high-weighted field; t1-t4, etc each occur twice, t5 once, their/time occur once
         // 'the' and 'to' occur too many times and get excluded
-        assertEquals(TITLE, suggestion.getAlternatives().get(0));
-        for (int i = 1; i <= 5; i++) {
+        // assertEquals(TITLE, suggestion.getAlternatives().get(0));
+        for (int i = 1; i <= 4; i++) {
             String sugg = suggestion.getAlternatives().get(i);
-            assertTrue(sugg + " does not match t[1-5]", sugg.matches("t[1-5]"));
+            assertTrue(sugg + " does not match t[1-5]", sugg.matches("t1[1-5]"));
         }
-        assertTrue(suggestion.getAlternatives().get(6).matches("their|time"));
-        assertTrue(suggestion.getAlternatives().get(7).matches("their|time"));
-        assertNotEquals(suggestion.getAlternatives().get(6), suggestion.getAlternatives().get(7));
+        suggestion = assertSuggestionCount("th", 2, "all");
+        assertTrue(suggestion.getAlternatives().get(1).matches("their"));
+        /*assertTrue(suggestion.getAlternatives().get(7).matches("their|time"));
+        assertNotEquals(suggestion.getAlternatives().get(6), suggestion.getAlternatives().get(7)); */

```

Comment:
I think this line was checking their|time because t1 could be matched with time. It is probably fine that we are not looking for time any more but maybe we should figure out what the equivalent to these asserts are.

Reply:
That's just because it led off from a single 't' before